src/Farmer/Builders/Builders.WebApp.fs

```diff
@@ -364,6 +366,16 @@ type WebAppConfig =
                   Tags = this.Tags}
             | _ ->
                 ()
+
+            yield!
```

Comment:
If you replace the `List.map` with a `for x in this.SiteExtensions do` then the `yield!` can be omitted.